model-optimizer/extensions/front/disable_weights_quantize_value_propagation.py

```diff
@@ -22,7 +22,6 @@ class DisableQuantizeValuePropagation(FrontReplacementPattern):
     enabled = True
     graph_condition = [
         lambda graph: graph.graph['cmd_params'].keep_quantize_ops_in_IR
```

Comment:
If we remove blobs_as_inputs, then keep_quantize_ops_in_IR should go too, as we do not have statistics section in V10 IR structure.